test/native/__mocks__/styleMock.js

```diff
@@ -106,4 +106,10 @@ module.exports = {
 		paddingLeft: 10,
 		paddingRight: 10,
 	},
+	'block-types-list__column': {
+		padding: 10,
+	},
+	floatingToolbar: {
+		height: 10,
+	},
```

Comment:
We're [mocking the CSS modules](https://github.com/WordPress/gutenberg/blob/f3f931bd68efa0e60e09a9b8d0e7dd2464c05041/test/native/jest.config.js#L51-L52) so if any object uses style values for calculations on their render functions, we have to add them here. In this case, I added a couple that I noticed that were required when rendering the editor and the inserter menu.

Reply:
I'm not very convinced to change the code for testing purposes, I always found that a bit anti-pattern. In this case, if it's only a matter of adding a small number of styles maybe it's still worth it to keep this static mock 🤔 , wdty?

Reply:
I do not have a strong opinion. While I agree that changing code for testing purposes is an anti-pattern, I think this `styleMock.js` file is a very awkward testing infrastructure to maintain. 

I do generally prefer to place "oddities" into the test files, but it is unfortunate that the oddity is so far reaching in regards to which tests it impacts. I.e. tests for many components will require one-off additions to this file for them to pass. 

I think I am happy to keep your current approach.

Reply:
>I think this styleMock.js file is a very awkward testing infrastructure to maintain.

I totally agree, I was a bit confused at the beginning about what was the purpose of this file 😄 .

>I do generally prefer to place "oddities" into the test files, but it is unfortunate that the oddity is so far reaching in regards to which tests it impacts. I.e. tests for many components will require one-off additions to this file for them to pass.

Good point, I'm going to note this as a topic to discuss internally because I don't have a strong opinion either.

>I think I am happy to keep your current approach.

Ok, let's keep this approach for now.